deployment-sidecar.yaml

```diff
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   name: vertx-create-span
   annotations:
-    inject-jaeger-agent: "true"
+    "sidecar.jaegertracing.io/inject": "true"
```

Comment:
are you sure the quotes are allowed in the key?

Reply:
The quotes are allowed, but not necessary. I'll remove them.